{{ cookiecutter.__repo_name }}/{{ cookiecutter.__app_name }}/config.py

```diff
@@ -17,28 +17,19 @@
 API_NAME = "{{ cookiecutter.__app_name }}"
 PROJECT_METADATA = metadata.metadata(API_NAME)  # .json
 
-# Fix metadata for emails from pyproject parsing
-_EMAILS = PROJECT_METADATA["Author-email"].split(", ")
-_EMAILS = map(lambda s: s[:-1].split(" <"), _EMAILS)
-PROJECT_METADATA["Author-emails"] = dict(_EMAILS)
-
-# Fix metadata for authors from pyproject parsing
-_AUTHORS = PROJECT_METADATA.get("Author", "").replace(", ", ",").split(",")
-_AUTHORS = [] if _AUTHORS == [""] else _AUTHORS
-_AUTHORS += PROJECT_METADATA["Author-emails"].keys()
-PROJECT_METADATA["Authors"] = sorted(_AUTHORS)
-
 # Get ai4-metadata.yaml metadata
-AI4_METADATA_DIR = os.getenv(
-    f"{API_NAME.capitalize()}_AI4_METADATA_DIR",
-    default=f"{os.getcwd()}/../{API_NAME}",
-)
+CWD = os.getcwd()
```

Comment:
This checks "ai4-metadata.yml" in:
 - $cwd/name_of_module
 - $cwd/../name_of_module which should be eq to run deepaas in any of the modules folders